docs/specification.md

```diff
@@ -121,13 +122,12 @@ target. gittuf applies this namespaced delegations graph traversal to Git and
 also incorporate RSLs and Git's implicit change tracking mechanisms.
 
 In gittuf, the delegations graph is similarly traversed, except that it
-explicitly does not expect any Targets metadata to contain an entry. Instead,
+explicitly does not expect any Targets metadata to contain a target entry. Instead,
 the delegation mechanism is used to identify the set of keys authorized to sign
 the target such as an RSL entry or commit being verified. Therefore, the
-delegation graph is searched until a delegation is encountered such that no
-metadata exists in the repository for the delegatee role. At this point, the
-search is terminated and the keys listed in that delegation entry are used as
-the set of authorized keys.
+delegation graph is used to decide which keys git actions should trust, but no
+targets entries are used.  Any key which delegated trust up to this part of the 
```

Comment:
I think I'm okay with this, it likely simplifies the implementation of delegation tree traversal.

Reply:
okay, great.  It helps to get rid of the weird case where Alice wants to keep control while delegating to Bob and whomever Bob wants, but Bob hasn't listed Alice.  Alice could remove the delegation, but this isn't ideal.